### ISSUE TITLE
No detailed error visible while excecuting

### ISSUE THREAD

**Issue by aperkivskyi** [crimdon/SQLToolkit#77]
While execution there is no detailed error message.
### use case:
**in the sql file i have (selecting not existing column)**
```
select 
       a1, a2, _notExistColumn_
from 
       dbo.Table
```
**In the log i can see**
`2020-05-04T09:48:05.7313337Z ##[error]Error running SQL script false`

Which is not enough for quick action (fix).

Desired behavior is to have option, to see raw message from the Sql Server, it will help to fix issue quickly.

**Comment by SaquibAlam** [comment]
Hello,
We are concerned with the same issue. The missing details in error log makes it difficult and time consuming to find and fix the errors.
Any chance to get details included soon?

**Comment by DuckoDeath** [comment]
agreed, this is a show-stopper, love this toolkit, but without the actual error being shown, going to have to look for something else.
